cdc/capture.go

```diff
@@ -55,7 +56,15 @@ func NewCapture(pdEndpoints []string) (c *Capture, err error) {
 		Endpoints:   pdEndpoints,
 		DialTimeout: 5 * time.Second,
 		DialOptions: []grpc.DialOption{
-			grpc.WithBackoffMaxDelay(time.Second * 3),
+			grpc.WithConnectParams(grpc.ConnectParams{
+				Backoff: backoff.Config{
+					BaseDelay:  time.Second,
+					Multiplier: 0.1,
```

Comment:
This should be greater than 1, otherwise the delay would become smaller and smaller and never reach the `MaxDelay`?